gems/aws-sdk-core/lib/seahorse/client/h2/connection.rb

```diff
@@ -10,56 +10,49 @@ module Seahorse
   module Client
     # @api private
     module H2
-
       # H2 Connection build on top of `http/2` gem
-      # (requires Ruby >= 2.1)
-      # with TLS layer plus ALPN, requires:
-      # Ruby >= 2.3 and OpenSSL >= 1.0.2
       class Connection
-
-        OPTIONS = {
-          max_concurrent_streams: 100,
-          connection_timeout: 60,
-          connection_read_timeout: 60,
-          http_wire_trace: false,
-          logger: nil,
-          ssl_verify_peer: true,
-          ssl_ca_bundle: nil,
-          ssl_ca_directory: nil,
-          ssl_ca_store: nil,
-          enable_alpn: false
-        }
+        OPTIONS = [
```

Comment:
Are there backwards/forwards risks in removing the defaults from here? 
IE - old service gem + new core? I think because the changes to async base client are also in core, we should be good, but just want to check.

Reply:
I don't think so. It's all contained in the connection/base.